index.js

```diff
@@ -12,21 +13,33 @@ const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
 const maxPage = 1;
-const page = 1;
-const searchQuery = "";
+let page = 1;
+const searchQuery = { value: "" };
 
 async function fetchCharacters() {
   try {
-    const response = await fetch("https://rickandmortyapi.com/api/character/");
+    const response = await fetch(
+      `https://rickandmortyapi.com/api/character/?name=${encodeURIComponent(
+        searchQuery.value
```

Comment:
here you can maybe think of leaving out the .value when you define let search query = ""